### DISCUSSION TITLE
如何使用返回按钮？

### DISCUSSION THREAD

**Discussion by xsx4525** [gkd-kit/gkd#807] [Q&A] [answered]
b站分享这个页面只能侧滑返回，如何解决？
![Screenshot_2024-12-06-22-29-44-929_com tencent mm](https://github.com/user-attachments/assets/868730b5-e9da-4652-9710-b1967a4ca398)

**Comment by lisonge** [comment, marked as answer]
https://gkd.li/api/interfaces/RawRuleProps#action

**Reply by xsx4525** [reply]
谢谢大佬
